packages/node/rollup.config.js

```diff
@@ -1,9 +1,11 @@
 import typescript from "rollup-plugin-typescript2";
 
 import pkg from "./package.json";
+
 const globals = {
-  ethers: "ethers",
-};
+  "@counterfactual/cf.js": "cfjs",
+  "eventemitter3": "EventEmitter3"
```

Comment:
@cf19drofxots I think this one is `EventEmitter`.

Reply:
Yeah, that's because the UMD version does that. We'd arguably use a different version without that extra namespace.

Reply:
Since there are no consumers yet to care about this, I'll change this to `EventEmitter`